installer/configurators/jenkins.py

```diff
@@ -0,0 +1,78 @@
+import subprocess
+import os
+import sys
+from pathlib import Path
+from installer.configurators.common import get_installer_root, get_tfvars_file, replace_tfvars
+
+jenkins_pem = get_installer_root() + "/jenkinskey.pem"
+
+
+def check_jenkins_pem():
+    """
+        Check if the user has provided jenkinskey.pem
+    """
+    print(
+        " Please make sure that you have the ssh login user names of jenkins and bitbucket servers."
+    )
+    # Check if user dropped the jenkins key into the installer root (scenario 1 only)
+    if not Path.is_file(jenkins_pem):
+        sys.exit(
+            "File jenkinskey.pem not found in installer root directory, kindly add and run the installer again! "
+        )
+
+    # Make sure the PEM the user dropped in has the right perms
+    subprocess.call('sudo chmod 400 {0}'.format(jenkins_pem))
```

Comment:
We might need shell=True.

Reply:
I'm dropping the `sudo`, it's pointless.

If the user running this script isn't already the owner of the file, running `sudo chmod 400` will literally prevent them from using it and the installer will fail anyway because of bad Unix perms.

If the user running this script IS the owner of the file, they don't need the `sudo`.

Reply:
ok. Getting this error, FileNotFoundError: [Errno 2] No such file or directory: 'chmod 400.....

Tried using subprocess.call('chmod 400 {0}'.format(jenkins_pem),shell=True), then the error gone.

Reply:
Oh, duh I'm an idiot. `call` expects a string array where the first arg is the executable. Fixed.